data_processing/pathology/common/preprocess.py

```diff
@@ -209,6 +282,25 @@ def pretile_scoring(slide_file_path: str, output_dir: str, params: dict):
     df.loc[:, "otsu_score"  ] = get_otsu_scores   (df['coordinates'], otsu_thumbnail, thumbnail_tile_size)
     df.loc[:, "purple_score"] = get_purple_scores (df['coordinates'], rbg_thumbnail,  thumbnail_tile_size)
 
+
+    # get pathology annotations for slide only if valid parameters
+    if slideviewer_dmt != None and slideviewer_dmt != "" and labelset != None and labelset != "":
+        annotation_url = os.path.join("http://", os.environ['MIND_API_URI'], "mind/api/v1/getPathologyAnnotation/", slideviewer_dmt, image_id, "regional", labelset)
+        logger.info(annotation_url)
+
+        response = urlopen(annotation_url)
+        response_text = response.read().decode(response.headers.get_content_charset())
```

Comment:
If you feel up to it, using requests might simplify this code block:

```
response = requests.get(route)
annotation_geojson = response.json()
```

Reply:
yes, this is way easier :)